lib/core/presentation/pages/event/guest_event_detail_page/widgets/host_event_basic_info_card.dart

```diff
@@ -0,0 +1,201 @@
+import 'dart:ui';
+
+import 'package:app/core/domain/event/entities/event.dart';
+import 'package:app/core/domain/event/entities/event_ticket.dart';
+import 'package:app/core/domain/event/input/get_tickets_input/get_tickets_input.dart';
+import 'package:app/core/domain/event/repository/event_ticket_repository.dart';
+import 'package:app/core/failure.dart';
+import 'package:app/core/presentation/widgets/image_placeholder_widget.dart';
+import 'package:app/core/presentation/widgets/theme_svg_icon_widget.dart';
+import 'package:app/core/utils/auth_utils.dart';
+import 'package:app/core/utils/event_utils.dart';
+import 'package:app/gen/assets.gen.dart';
+import 'package:app/gen/fonts.gen.dart';
+import 'package:app/i18n/i18n.g.dart';
+import 'package:app/injection/register_module.dart';
+import 'package:app/theme/spacing.dart';
+import 'package:app/theme/typo.dart';
+import 'package:cached_network_image/cached_network_image.dart';
+import 'package:dartz/dartz.dart';
+import 'package:duration/duration.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+
+class HostEventBasicInfoCard extends StatelessWidget {
```

Comment:
lets move these widgets to another folder 
/pages/event/host_event_detail_page/widgets